openfisca_france/parameters/prelevements_sociaux/reductions_cotisations_sociales/allegement_cotisation_allocations_familiales/plafond_smic_2023_12_31.yaml

```
@@ -0,0 +1,18 @@
+description: Plafond de rémunération (en nombre de Smic du 31 décembre 2023) des salariés concernés par l'allègement des cotisations pour la branche famille des cotisations de Sécurité sociale (SS)
+values:
+  2024-01-01:
+    value: 3.5
```

```suggestion
    value: 2.5
```

@clallemand ce n'est pas 2,5 ?